aepp/utils.py

```diff
@@ -0,0 +1,25 @@
+from configparser import ConfigParser
+import os
+
+class Utils:
+
+    global config_path
+
+    def __init__(self):
+        self.config_path = os.path.join(os.environ["ADOBE_HOME"], "conf", "config.ini")
```

Comment:
I would offer to pass the environment variable as a parameter of the class instanciation. 
Do not assume that there is a config.ini file or an environment variable available

Reply:
We have briefly discussed the question while reviewing the design wiki. https://wiki.corp.adobe.com/pages/viewpage.action?pageId=2931736486, see the first open question in section 5 . Since the initial purpose for adding utils method is to help one of the CME customer to save their entity ids, we do request CME customer to set up the config.ini and environment variable ADOBE_HOME. But I did see @pitchmuc 's point on this, WDYT? @cmenguy

Reply:
I would agree with @pitchmuc - I know that for `ADOBE_HOME` we used that for CMLE but here it's never been mentioned in this repo. If we want to standardize with having this variable setup to find configs I think it will need a larger effort to do that in a more streamlined way (like what for example the AWS CLI does), and also this will need to be added to documentation. For now I would remove it until we have a clear plan to support this env variable.